BLSQ/iaso
# IA-2679 PLANNING refactor

Initial demand was to review the process of  parent picking mode for the list view.

Finally i reviewed the whole page to make it more easy to understand

Related JIRA tickets : IA-2679

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- pass org unit type id and name while getting an org unit with parents
- use org unit type id and name in list columns
- remove parent picking option
- split way to select parent in map view and list view (dropdown on map and check box on list)
- highlight column when came org unit type is selected
- move tabs at a lower level

## How to test

Open a planning, drill down to a lower level org unit (village, hopital)
Open list view and check one of the column, click on a row, you should be able to assign all the org unit in the org unit type selected to the current team / user.
Check the same behaviour on the map

## Print screen / video
[
Upload here print screens or videos showing the changes
](https://github.com/BLSQ/iaso/assets/12494624/8f170805-9bf1-4742-ab9e-f1f7cc3c642b)
## Notes

- right click should be removed and replaced by a sticky tooltip => https://bluesquare.atlassian.net/browse/IA-2685
- on a sortable column, when you leave a your mouse on a head cell, a title appears saying you can sort this column, even if you are hovering a button or a checkbox in this case
